products/apigateway/terraform.yaml

```diff
@@ -43,6 +44,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       managedService: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
   ApiConfig: !ruby/object:Overrides::Terraform::ResourceOverride
+    docs: !ruby/object:Provider::Terraform::Docs
+      optional_properties: |
+        * `api_config_id_prefix` - (Optional) Creates a unique name beginning with the
+         specified prefix. Conflicts with `api_config_id`.
```

Comment:
```suggestion
         specified prefix. Either this or `api_config_id` must be specified.
```

Reply:
"If this and `api_config_id` are unspecified, a random value is chosen for the name."

Reply:
fixed